### DISCUSSION TITLE
Renko Chart

### DISCUSSION THREAD

**Discussion by DaveSkender** [DaveSkender/Stock.Indicators#478] [Indicators]
Community Discussion about [Renko Charts](https://dotnet.stockindicators.dev/indicators/Renko/#content) price transform

**Comment by shinel70** [comment]
i suggest to fix GetRenko.
with brick size it must start from 0 price not the first quote we provide.
and for GetRenkoAtr to not use last ATR, my suggest is to use first or last or average of ATR that pieace of candlesticks; example with new RenkoCandle GetRenkoAtr consider first or last ATR or Candlestick (or Average of CandleStick'Atr)

**Reply by DaveSkender** [reply]
This sounds more like an alternate approach than a fix.  We’re using the [standard method](https://school.stockcharts.com/doku.php?id=chart_analysis:renko).

**Reply by shinel70** [reply]
I tested your Renko and it seems it is not standard as written in [school.stckcharts.com](https://school.stockcharts.com/doku.php?id=chart_analysis:renko) "They move up or down in 45-degree lines with one brick per vertical column."
the problem start when the trend reverse the next (red in uptrend and green in downtrend) candles build side by side before one not in 45-degree.

**Reply by DaveSkender** [reply]
the concept of 45-degrees is based on how you setup your visualization, and is not directly something we can control from the library.  can you elaborate on the reversal aspect?  I’m not clear on what you’re explaining.

**Reply by shinel70** [reply]
here is a picture i try to explain my problem 
![get renko](https://user-images.githubusercontent.com/46675285/169690633-85fde08d-c41c-4d81-8496-16d6003dff4a.jpg)
`
using Binance.Net.Clients;
using Skender.Stock.Indicators;
BinanceClient bc = new();
var data = await bc.UsdFuturesApi.CommonFuturesClient.GetKlinesAsync("BTCUSDT", TimeSpan.FromMinutes(1));
List<Quote> quotes = data.Data.Select(k => new Quote
{
    Open = k.OpenPrice.Value,
    High = k.HighPrice.Value,
    Close = k.ClosePrice.Value,
    Low = k.LowPrice.Value,
    Date = k.OpenTime,
    Volume = k.Volume.Value
}).ToList();
var renkoResult = quotes.GetRenko(10);
Console.ReadLine();`

**Reply by DaveSkender** [reply]
Ah, yes, this is a problem.  I've created issue #812 to further investigate and fix.  Thanks for reporting!
